### PR TITLE
Implement `getChannelName` remote function

### PR DESCRIPTION
**Pull request by LakshanSS**
### Description
- Implement `getChannelName` remote function
- Fix https://github.com/ballerina-platform/ballerina-extended-library/issues/340

### CODE REVIEW

File: slack/endpoint.bal [hunk 1]
```diff
@@ -197,6 +197,14 @@ public isolated client class Client {
...
 
+    # Gets name of a channel by id.
+    # 
+    # + channelId - Id of the channel
+    # + return - Name of the channel if it is a success or an error if it is a failure
+    remote isolated function getChannelName(string channelId) returns string|error {
+        return check getChannelName(self.slackClient, channelId);
+    }
+
     // User specific functions
```
Comment (SanduDS) [inline slack/endpoint.bal:209]: Shall we need these empty lines?

Comment (LakshanSS) [reply slack/endpoint.bal:209]: Yes. We need this.
This comment is used to categorise remote function

File: slack/utils.bal [hunk 2]
```diff
@@ -134,6 +134,22 @@ isolated function getConversationInfo(http:Client slackClient, string channelId,
...
+    http:Response response = check slackClient->get(LIST_CONVERSATIONS_PATH);
+    json channelList = check response.getJsonPayload();
+    map<json> channelListJson = <map<json>> channelList;
+    _ = check checkOk(channelListJson);
+    json channels = check channelListJson.channels;
+
+    json[] channelsArr = <json[]> channels;
+    foreach json channel in channelsArr {
+        if channel.id === channelId {
+            return <string> check channel.name;
```
Comment (kapilraajP) [inline slack/utils.bal:147]: Can't we use `ensureType()` here instead of converting to string like this?

Comment (kapilraajP) [inline slack/utils.bal:147]:
```suggestion
            return check channel.name.ensureType(string);
```

Comment (LakshanSS) [reply slack/utils.bal:147]: +1

File: slack/utils.bal [hunk 3]
```diff
@@ -134,6 +134,22 @@ isolated function getConversationInfo(http:Client slackClient, string channelId,
...
+isolated function getChannelName(http:Client slackClient, string channelId) returns string|error {
+    http:Response response = check slackClient->get(LIST_CONVERSATIONS_PATH);
+    json channelList = check response.getJsonPayload();
+    map<json> channelListJson = <map<json>> channelList;
+    _ = check checkOk(channelListJson);
+    json channels = check channelListJson.channels;
+
+    json[] channelsArr = <json[]> channels;
+    foreach json channel in channelsArr {
+        if channel.id === channelId {
```
Comment (SanduDS) [inline slack/utils.bal:146]: Can't we use `==` here?

File: slack/utils.bal [hunk 4]
```diff
@@ -134,6 +134,22 @@ isolated function getConversationInfo(http:Client slackClient, string channelId,
     }
 }
 
+isolated function getChannelName(http:Client slackClient, string channelId) returns string|error {
+    http:Response response = check slackClient->get(LIST_CONVERSATIONS_PATH);
+    json channelList = check response.getJsonPayload();
+    map<json> channelListJson = <map<json>> channelList;
+    _ = check checkOk(channelListJson);
+    json channels = check channelListJson.channels;
+
```
Comment (SanduDS) [inline slack/utils.bal:143]: Empty line?

Comment (LakshanSS) [reply slack/utils.bal:143]: This is used to separate the code as there are many lines of code within that function.

File: slack/utils.bal (hunk 3 above)
Comment (LakshanSS) [reply slack/utils.bal:146]: +1

File: slack/utils.bal (hunk 2 above)
Comment (LakshanSS) [reply slack/utils.bal]: Done

**Review by kapilraajP — APPROVED** [review]

**Pull request merged**